Dockerfile

```diff
@@ -5,9 +5,7 @@ RUN git config --global --add safe.directory /src
 RUN npm install
 RUN npm install -g netlify-cli
 
-RUN python3 -m venv .venv
-RUN . .venv/bin/activate
-RUN pip3 install -r requirements.txt
+RUN pip3 install -r requirements.txt --break-system-packages
```

Comment:
Isn't using `venv` the recommended approach? Is there a problem with a recent update to the hugo image that requires this now?

Reply:
Last updated June 6, 2023… so not that. Something in the OS package repositories?

It might be worth a quick comment in the Dockerfile explaining why this approach is needed in case someone tries changing it back to venv in future.

Reply:
@AlanGreene - uhm, I don't remember doing this change, neither why I would remove the venv... the docker image is only there to support development, but tbh I never use it. Let me re-spin with PR without changes to the Dockerfile.